src/mumble/GlobalShortcut_win.h

```diff
@@ -82,6 +82,16 @@ class GlobalShortcutWin : public GlobalShortcutEngine {
 		static LRESULT CALLBACK HookKeyboard(int, WPARAM, LPARAM);
 		static LRESULT CALLBACK HookMouse(int, WPARAM, LPARAM);
 
+		/// Handle an incoming Windows keyboard message.
+		///
+		/// Returns true if the button was suppressed, otherwise returns false.
```

Comment:
Well, whether or not a button is suppressed is handled when the GlobalShortcut engine processes the button. That's why I wrote it like this. It fits my mental model of what happens.

Maybe it should read: "Returns true if the button was suppressed by the GlobalShortcut engine, otherwise returns false." instead?